mods/example/audio/notifications.yaml

```diff
@@ -1,8 +0,0 @@
-Speech:
-	DefaultVariant: .wav
-	Notifications:
```

Comment:
I added this on purpose in https://github.com/OpenRA/OpenRAModSDK/pull/133 as not having them will crash the game when the mod gets expanded on. The structure is very much hard-coded in multiple locations of the engine.

Reply:
Then i would suggest to fix this in the engine. Having a default of .wav is nice but instead one could simply add .wav to the filenames. Notifications however is an empty collection. Making them that hardcoded that mods need those node defined without any purpose is just hacky. Then i would as said above prefer to fix that crashes in the engine or make the loader add those node per default if not present.

Reply:
> Then i would suggest to fix this in the engine.
👍 

We need to address this in the nearish term in order to support the remastered assets in RA and TD. My current thoughts are that we move the whole thing into mod code, like with sequences.